web/srv/server.go

```diff
@@ -197,6 +201,15 @@ func NewServer(
 	server.router.POST("/grafana/*grafanapath", handler.handleGrafana)
 	server.router.PUT("/grafana/*grafanapath", handler.handleGrafana)
 
+	// jaeger proxy
```

Comment:
Should there be a helper function that manages the paths for both grafana and jaeger? It is a minor optimization, my DRY senses are tingling though.

Reply:
Yep, Done!